utilities/data_collector.py

```diff
@@ -78,6 +80,18 @@ def collect_alerts_data():
     )
 
 
+def collect_vnc_screenshot_for_vms() -> None:
```

Comment:
since it's called inside `wait_for_running_vm`, why not pass vm as param here?

Reply:
It would create circular dependency. But I can pass the vm name and namespace.